compose/bin/check-dependencies

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+declare -A common_dependencies
```

Comment:
Should this be: `declare -a common_dependencies` ?

Reply:
Yes, you're right.

Thx, I fixed it!

Reply:
I didn't get an answer to this code review -- should this be `declare -a common_dependencies` ?